12_glob_lab/globlab.txt

```diff
@@ -1,29 +1,59 @@
 Glob Lab.
 https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
-1. Type in a glob pattern to match:
+1. 
+
+Type in a glob pattern to match:
 
-  1. All files whose name starts with a lowercase letter
+ 
+ 1. All files whose name starts with a lowercase letter
   Answer: 
 
-  2. All files whose name starts with an uppercase letter
+ 
+ls -d [[:lower:]]*/*
+ 2. All files whose name starts with an uppercase letter
   Answer:
 
-  3. All files whose name ends in an "a", "e", "i", "o" or "u"
+ 
+ls -d [[:upper:]]*/* - for some reason shows the following:
+Doc/README.txt Doc/test.c
+Why test.c as well?

```

Comment:
The problem is with the forward slash `/`:

Your glob pattern matches all the files in all directories which (the directory) start with an uppercase letter.

Better to skip the second part of the glob and leave it at:

```
ls -d [[:upper:]]*
```

Or, if you also want to see the contents of each directory drop the d:

```
ls [[:upper:]]*
```

Alternatively, showing all the FILES that start with an uppercase in all directories would look like this:

```
ls -d */[[:upper:]]*
```